OSDOCS-13305: update image mode docs for GA MicroShift (#92407)
openshift/openshift-docs: modules/microshift-install-bootc-conc.adoc

[id="microshift-bootc-conc_{context}"]
= About image mode for {op-system-base-full}

> By using image mode for {op-system-base-full} {op-system-base}, you can administer the operating system just as you would any other container. Image mode for {op-system-base} is a deployment method that uses a container-native approach to build, deploy, and manage the operating system as a bootc image.

Reviewer: ```suggestion
By using image mode for {op-system-base}, you can administer the operating system just as you would any other container. Image mode for {op-system-base} is a deployment method that uses a container-native approach to build, deploy, and manage the operating system as a bootc image.
```

* This container image uses standard OCI or Docker containers as a transport and delivery format for base operating system updates.
* A bootc image includes a Linux kernel that is used to start the operating system.
* By using bootc containers, developers, operations administrators, and solution providers can all use the same container-native tools and techniques.

Image mode for {op-system-base} splits the creation and installation of software changes into two steps: one on a build system and one on a running target system.

> * In the build-system step, a Podman build inspects the RPM files available for installation, determines any dependencies, and creates an ordered list of chained steps to complete. The end result is a new operating system available to install.

Reviewer 2: I'm not sure about this paragraph. While it's true for RPM-related steps of Podman build, there may also be other system configuration steps taking place. Can this be reflected somehow in the sentence?

> * In the running-target-system step, a bootc update downloads, unpacks, and makes the new operating system bootable alongside the currently running system. Local configuration changes are carried forward to the new operating system. These changes only take effect when the system is rebooted and the new operating system image replaces the previously running one.

Reviewer: IBM Style lists "boot" as a verb as a term to avoid: https://www.ibm.com/docs/en/ibm-style?topic=word-usage#boot

Maybe this would work?
```suggestion
* In the running-target-system step, a bootc update downloads the operating system, unpacks it, and prepares it to be started alongside the currently running system. Local configuration changes are carried forward to the new operating system. These changes take effect only when the system is restarted and the new operating system image replaces the previously running one.
```
